### DISCUSSION TITLE
Help with CloudlogCAT

### DISCUSSION THREAD

**Discussion by m0sdb** [magicbug/Cloudlog#3020] [Q&A] [answered]
Hi all

Fairly new to Cloudlog and I am struggling to get CAT control to work.  I have set up Omnirig (and Omnirig has been working just FB for ages with other logging software) and CloudCAT is confirming the radio is on-line.

I have then gone into the API bit and for certainty used the little copy button to copy both the URL and API key (which is a read/write one) and pasted these into Cloud log bit in the relevant CloudlogCAT settings and...

It tells me sync has failed and to check the URL/API both of which I have.  I have also tested the API and this appears to be working fine also.

Am I missing something?  Note it is a Cloudlog sub hosted instance I am using.

Thanks all

Danny M0SDB

**Comment by magicbug** [comment]
I'll ask the stupid questions first

Do you see frequency data in CloudlogCAT? 

With the URL make sure its just the URL without any trailing / on the end

**Reply by m0sdb** [reply]
Hmm - thanks for getting back so quick.  Odd, after a reboot it is now syncing with Cloudlog so that is good.

Not so good is that whilst the rig is showing as online, it is not pulling the frequency data through.  I have just tested it with LogHX3 and that is pulling the frequency through using the same Omnirig instance (Omnirig v1.20) so I do not think it is an Omnirig issue.

**Reply by m0sdb** [reply]
I have checked as well and CloudlogCAT is pulling through the mode just fine and it changes as I change it on the radio so it is talking to Omnirig - it is just not pulling through the frequency data.

I have checked also and the mode does pull through to Cloudlog itself.  So just the frequency data to resolve :)

**Reply by magicbug** [reply, marked as answer]
icom radio? you might need to grab https://github.com/magicbug/CloudlogCAT/releases/download/1.8/CloudlogCAT.Test.Release.ICOM.zip

**Reply by m0sdb** [reply]
Brill - thanks Peter.  That does indeed work with a Ten-Tec Orion 565.  That is excellent :)

**Reply by magicbug** [reply]
Phew! :)

**Comment by m0sdb** [comment]
It is a Ten-Tec Orion but I will try that in case it mirrors Icom
